buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -236,7 +236,7 @@
       "inputs": {
         "scriptType": "inlineScript",
         "scriptName": "",
-        "arguments": "-ConfigurationGroup $(PB_ConfigurationGroup) -OutputFeedUrl $(PB_AzureSymbolFeedUrl) -StorageAccountKey $(PB_PublishAccountKey)",
+        "arguments": "-ConfigurationGroup $(PB_ConfigurationGroup) -OutputFeedUrl $(PB_PublishBlobFeedSymbolUrl) -StorageAccountKey $(PB_PublishBlobFeedKey)",
         "workingFolder": "$(Pipeline.SourcesDirectory)",
         "inlineScript": "param($ConfigurationGroup, $OutputFeedUrl, $StorageAccountKey)\n msbuild src\\publish.proj /t:PublishSymbolsToAzureBlobFeed /p:AccountKey=$StorageAccountKey /p:ExpectedFeedUrl=$OutputFeedUrl /p:ConfigurationGroup=$ConfigurationGroup /p:PublishSymbols=\"true\"",
         }
```

Comment:
Need to change the condition on version repo to `and(succeeded(), ne(variables.PB_PublishType, ''), eq(variables['PB_ConfigurationGroup'], 'Release'))`